google_drive/public/css/google_calendar.css

```diff
@@ -0,0 +1,14 @@
+.google-calendar-xblock-wrapper{
```

Comment:
I can't really comment much on the CSS -- perhaps @marcotuts would be a good person to review this aspect of the block?